src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -487,8 +495,8 @@ void LiveFunctionsDataView::OnTimer() {
 
 void LiveFunctionsDataView::OnRefresh(const std::vector<int>& visible_selected_indices,
                                       const RefreshMode& mode) {
-  if (mode != RefreshMode::kOnFilter && mode != RefreshMode::kOnSort) return;
-  UpdateHighlightedFunctionId(visible_selected_indices);
+  if (mode != RefreshMode::kOther) UpdateHighlightedFunctionId(visible_selected_indices);
```

Comment:
You can set if statements in one line (go/cstyle#Conditionals), but I will separate them at least with an enter.